[2022-08-13 12:19:38,762] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.convert_xls_into_xlsx manual__2022-08-13T12:19:31.927138+00:00 [queued]>
[2022-08-13 12:19:38,776] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.convert_xls_into_xlsx manual__2022-08-13T12:19:31.927138+00:00 [queued]>
[2022-08-13 12:19:38,777] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 12:19:38,777] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-13 12:19:38,778] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 12:19:38,799] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): convert_xls_into_xlsx> on 2022-08-13 12:19:31.927138+00:00
[2022-08-13 12:19:38,805] {standard_task_runner.py:52} INFO - Started process 3980 to run task
[2022-08-13 12:19:38,814] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'convert_xls_into_xlsx', 'manual__2022-08-13T12:19:31.927138+00:00', '--job-id', '350', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpcvzro9qx', '--error-file', '/tmp/tmpf7vaxito']
[2022-08-13 12:19:38,815] {standard_task_runner.py:80} INFO - Job 350: Subtask convert_xls_into_xlsx
[2022-08-13 12:19:38,902] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.convert_xls_into_xlsx manual__2022-08-13T12:19:31.927138+00:00 [running]> on host 4bbb34367c41
[2022-08-13 12:19:39,015] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=convert_xls_into_xlsx
AIRFLOW_CTX_EXECUTION_DATE=2022-08-13T12:19:31.927138+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-13T12:19:31.927138+00:00
[2022-08-13 12:19:39,017] {ETL_Raizen.py:47} INFO - Converting file .xls into .xlsx using libreoffice
[2022-08-13 12:19:42,222] {ETL_Raizen.py:49} INFO - File .xls succesfully converted into .xlsx
[2022-08-13 12:19:42,223] {python.py:173} INFO - Done. Returned value was: None
[2022-08-13 12:19:42,240] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=convert_xls_into_xlsx, execution_date=20220813T121931, start_date=20220813T121938, end_date=20220813T121942
[2022-08-13 12:19:42,283] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-13 12:19:42,339] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
